Source/Core/PolylineGeometry.js

```diff
@@ -309,9 +309,11 @@ define([
         var k;
 
         var positions = arrayRemoveDuplicates(polylineGeometry._positions, Cartesian3.equalsEpsilon);
-
         var positionsLength = positions.length;
-        if (positionsLength < 2) {
+
+        // A width of a pixel or less is not a valid geometry, but in order to support external data
+        // that may have errors we treat this as an empty geometry.
+        if (positionsLength < 2 || width < 1.0) {
```

Comment:
You might be right.  The Polyline primitive works with a width of 0.5, I don't see why the PolylineGeometry wouldn't.  @erikmaarten try changing this to <= 0 and see what happens.  Here's a code example so you can see what the Polyline primitive looks like:
``` javascript
var viewer = new Cesium.Viewer('cesiumContainer');
var collection = viewer.scene.primitives.add(new Cesium.PolylineCollection());
collection.add({
    positions : Cesium.Cartesian3.fromDegreesArrayHeights([-75, 35, 100000, -85, 35, 100000]),
    width : 0.5
});
```